src/math/lp/lp_core_solver_base.h

```diff
@@ -449,40 +452,40 @@ class lp_core_solver_base {
         out << "[" << j << "] " << std::setw(6) << " := " << j_val;
         if (m_basis_heading[j] >= 0)
             out << " base ";
-        else 
+        else
             out << "      ";
         for (auto k = j_val.size(); k < 15; ++k)
             out << " ";
         switch (m_column_types[j]) {
-        case column_type::fixed:
-        case column_type::boxed:
-            out << "[" << m_lower_bounds[j] << ", " << m_upper_bounds[j] << "]";
-            break;
-        case column_type::lower_bound:
-            out << "[" << m_lower_bounds[j] << ", oo" << "]";
-            break;
-        case column_type::upper_bound:
-            out << "[-oo, " << m_upper_bounds[j] << ']';
-            break;
-        case column_type::free_column:
-            out << "[-oo, oo]";
-            break;
-        default:
-            UNREACHABLE();
+            case column_type::fixed:
+            case column_type::boxed:
+                out << "[" << m_lower_bounds[j] << ", " << m_upper_bounds[j] << "]";
+                break;
+            case column_type::lower_bound:
+                out << "[" << m_lower_bounds[j] << ", oo"
+                    << "]";
+                break;
+            case column_type::upper_bound:
+                out << "[-oo, " << m_upper_bounds[j] << ']';
+                break;
+            case column_type::free_column:
+                out << "[-oo, oo]";
+                break;
+            default:
+                UNREACHABLE();
         }
         return out << "\n";
     }
 
     bool column_is_fixed(unsigned j) const { return this->m_column_types[j] == column_type::fixed; }
 
-    
     bool column_has_upper_bound(unsigned j) const {
```

Comment:
why is "case" indented? 
This is the first time I see an indented case relative to switch.